routes/admin.js

```diff
@@ -2,8 +2,53 @@ const express = require('express');
 const router  = express.Router();
 const pollsQueries = require('../db/queries/polls');
 const choiceQueries = require('../db/queries/choices');
+const voteQueries = require('../db/queries/votes');
 
 
+<<<<<<< HEAD
+router.get('/:id',(req,res)=>{
+  const link = req.params.id.split(':')[1];
+  console.log(link);
+  pollsQueries.getPollByLink(link)
+  .then((result)=>{
+    if(result){
+      if (link == result.voter_link){
```

Comment:
we can assume that it is the admin link, because this is the /admin route. This is just a demo and there doesn't need to be any checks for things like that right now.